core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -367,6 +368,53 @@ object SparkSubmit extends CommandLineUtils with Logging {
       }.orNull
     }
 
+    // When running in YARN cluster manager,
```

Comment:
"When running in YARN cluster manager, ?"

Reply:
Sorry for the broken comment, my bad, I will fix it.